pkg/rancher-desktop/config/settings.ts

```diff
@@ -86,13 +86,8 @@ export const defaultSettings = {
     name: ContainerEngine.MOBY,
   },
   virtualMachine: {
-    memoryInGB:   2,
-    numberCPUs:   2,
-    /**
-     * when set to true Dnsmasq is disabled and all DNS resolution
-     * is handled by host-resolver on Windows platform only.
-     */
-    hostResolver: true,
```

Comment:
This should have a version bump + migration. Probably a good time to also drop `.networkingTunnel`.